cleanup_b4_pull_request.md

```diff
@@ -0,0 +1,49 @@
+# Cleaning up history prior to creating a pull request
+The idea of cleaning up your branch's history before creating a pull request can be a controversial topic.
+Some project managers require some level of cleanup and some do not. Sometimes it is up to the developer to 
+decide how much (if any) cleanup is done before creating a pull request. Here we will discuss some of 
+the factors to consider when deciding how much (if any) clean up you may want to do before creating a
+pull request.
+We will also explain how to perform some of the cleanup options.
+This discussion assumes we are using in a fork/clone development workflow.
```

Comment:
> fork/clone development workflow

Can we link to this file? https://github.com/mindfulness-at-the-computer/git-info/blob/master/fork-clone-workflow.md

Reply:
Yes, but I was planning on filling in the instructions part before linking it in.
Are OK with merging this version (not linked in yet) while I work on the rest of the document?

Reply:
Alright, sounds good!